services/core-api/app/api/parties/party_appt/models/mine_party_appt.py

```diff
@@ -66,9 +66,16 @@ def save(self, commit=True):
         if commit:
             if not (self.permit or self.permit_id or self.mine_guid or self.mine):
                 raise AssertionError("Must have a related permit or mine")
-            if self.mine_party_appt_type_code in PERMIT_LINKED_CONTACT_TYPES and (
-                    self.mine_guid or self.mine) is not None:
-                raise AssertionError("Permittees are not related to mines")
+
+            if self.mine_party_appt_type_code == 'PMT' and (self.mine_guid
+                                                            or self.mine) is not None:
+                raise AssertionError("Contacts linked to a permit are not related to mines")
+
```

Comment:
How does a user get this error via the UI?

Reply:
This is a toast message, in theory, the user would not see this one, it is an additional step to prevent such cases when we have weird data.